code/modules/organs/external/_external.dm

```diff
@@ -365,44 +365,47 @@
 	return all_limbs
 
 /obj/item/organ/external/proc/is_dislocated()
-	if(dislocated > 0)
-		return 1
-	if(is_parent_dislocated())
-		return 1//if any parent is dislocated, we are considered dislocated as well
-	return 0
+	return (dislocated > 0) || is_parent_dislocated() //if any parent is dislocated, we are considered dislocated as well
```

Comment:
Could just be `dislocated || is_parent_dislocated()` now.

Reply:
True